htlcswitch/switch_test.go

```diff
@@ -3273,12 +3283,30 @@ func TestSwitchHoldForward(t *testing.T) {
 		t:               t,
 		interceptedChan: make(chan InterceptedPacket),
 	}
-	switchForwardInterceptor := NewInterceptableSwitch(s, false)
+	const cltvRejectDelta = 13
+	switchForwardInterceptor := NewInterceptableSwitch(
+		s, cltvRejectDelta, false,
+	)
 	require.NoError(t, switchForwardInterceptor.Start())
 
 	switchForwardInterceptor.SetInterceptor(forwardInterceptor.InterceptForwardHtlc)
 	linkQuit := make(chan struct{})
 
+	// Test a forward that expires too soon.
+	ogPacket.incomingTimeout = testStartingHeight + 5
```

Comment:
nit: Instead of +5 and +20 you could reuse the `cltvRejectDelta`.

Reply:
done